zebrad/tests/acceptance.rs

```diff
@@ -631,6 +635,58 @@ fn sync_until(
     Ok(child.dir)
 }
 
+#[test]
+#[ignore]
+fn sync_to_sapling() -> Result<()> {
+    let mut testdir = testdir()?;
+    let backup_dir = dirs::cache_dir()
+        .ok_or_else(|| eyre!("no cache dir found"))?
+        .join("sapling_backup");
+
+    if !backup_dir.exists() {
+        testdir = create_sapling_backup(testdir, &backup_dir)?;
+    }
+
+    for entry in std::fs::read_dir(backup_dir)? {
+        let entry = entry?;
+        std::process::Command::new("cp")
+            .arg("-r")
+            .arg(entry.path())
+            .arg(testdir.path())
+            .status2()?
+            .assert_success()?;
+    }
+
+    let height = LARGE_CHECKPOINT_TEST_HEIGHT;
+    let network = Mainnet;
+    let timeout = Duration::from_secs(60);
+
+    println!("Running real sync");
+    sync_until(height, network, STOP_AT_HEIGHT_REGEX, timeout, testdir)?;
+
+    Ok(())
+}
+
+fn create_sapling_backup(testdir: TempDir, dest: &Path) -> Result<TempDir> {
+    println!("Creating backup database");
+    // 1 hour
+    let timeout = Duration::from_secs(60 * 60);
+    let network = Mainnet;
```

Comment:
Instead of hard-coding mainnet, can we pass a Network argument to these functions?

It would be good to implement the corresponding tests for testnet, even if we don't run them by default.